app/src/test/java/uk/ac/ebi/atlas/search/metadata/HighchartsSunburstAdapterTest.java

```diff
@@ -0,0 +1,110 @@
+                                species1OrganismPart2ExperimentAccession)),
+                        // Organism part 1
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName(), species1OrganismPart1),
+                                species1OrganismPart1ExperimentAccession)),
+                        // Organism part 2
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName(), species1OrganismPart2),
+                                species1OrganismPart2ExperimentAccession)),
+                        // Organism part 1, cell types
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType1),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType2),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType3),
+                                species1OrganismPart1ExperimentAccession)),
+                        // Organism part 2, cell types
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart2, species1OrganismPart2CellType1),
+                                species1OrganismPart2ExperimentAccession)));
+    }
+
+    @DisplayName("This unit test use the randomised input test data and test the functionality of the wheel")
+    @Test
+    void cellTypeWheelSunburstResultObjectShouldNotContainIdAndParentAttributeValuesSame() {
+        ImmutableSet<ImmutableMap<String, ?>> result = subject.getCellTypeWheelSunburst(metadataSearchTerm,cellTypeWheelServiceMock
+                .search(metadataSearchTerm, species1.getName()));
+        assertThat(result).hasSize(2);
+        assertThat(result.asList().get(0).get("id")).isNotEqualTo(result.asList().get(0).get("parent"));
+        assertThat(result.asList().get(1).get("id")).isNotEqualTo(result.asList().get(1).get("parent"));
+    }
+
+    @Test
+    @DisplayName("This test uses hardcoded mock test data produced by the public server and tests the pre and post-bugfix code precisely to make sure that bug will not reappear again." +
+            "Both unit tests look the same, the only difference is in the input test data.")
```

Comment:
Again, these are code comments. A unit test should’t mention other tests for understanding its meaning.